services/vod-to-live/service/src/domains/smil/generator/utils.ts

```diff
@@ -1,4 +1,5 @@
-import { CuePointSchedule, DetailedVideo } from '@axinom/mosaic-messages';
+import { DetailedVideo } from '@axinom/mosaic-messages';
```

Comment:
I think it would be more correct to change `DetailedImage` and `DetailedVideo`
to be from `media-messages` here and in all other places, since `media-messages`
now defines them and relates them to appropriate messages